chess/src/Components/Leaderboard.js

```diff
@@ -28,11 +23,11 @@ class Leaderboard extends React.Component {
   }
 
   render() {
-    console.log(this.state.data.leaderboard);
-    const leaderboard = this.state.data.leaderboard.map((person) => {
+    console.log(this.state.data);
+    const leaderboard = this.state.data.leaderboard.map((person, i) => {
```

Comment:
maybe this could go in a separate function?

Reply:
good idea